libs/build-all.sh

```diff
@@ -5,13 +5,13 @@ set -euvx
 OPENSSL_VERSION="1.1.1a"
 OPENSSL_SHA256="fc20130f8b7cbd2fb918b2f14e2f429e109c31ddd0fb38fc5d71d9ffed3f9f41"
 
-SQLCIPHER_VERSION="4.1.0"
-SQLCIPHER_SHA256="65144ca3ba4c0f9cd4bae8c20bb42f2b84424bf29d1ebcf04c44a728903b1faa"
+# SQLCIPHER_VERSION="4.1.0"
```

Comment:
bringing this back after the CI runs green?

Reply:
Once sqlcipher merges our NSS changes.